data/test_config/config_defaults.yml

```diff
@@ -23,3 +23,13 @@ pipeline: []
 #     ambiguity_threshold: 0.1
 
 data:
+policies:
```

Comment:
I think that's an unwanted change, isn't it?

Same as the `TEDPolicy_confusion_matrix.png` and `other`